pkg/git/copy_test.go

```diff
@@ -13,17 +13,23 @@ import (
 )
 
 func TestCopyService(t *testing.T) {
-	s := &mockSource{localPath: "/tmp/testing"}
-	files := []string{"services/service-a/base/config/my-file.yaml", "services/service-a/base/config/this-file.yaml"}
+	s := &mockSource{localPath: "/"}
+	files := []string{"/environments/dev/services/service-a/base/config/my-file.yaml", "/environments/dev/services/service-a/base/config/this-file.yaml"}
 	for _, f := range files {
 		s.addFile(f)
 	}
 	d := &mockDestination{}
 
-	copied, err := CopyService("service-a", s, d)
+	// They're the same because this is only testing the copying picks up all the files
+	// The promotion part is done elsewhere
+	sourceEnvironment := "dev"
```

Comment:
Probably no need to define these vars and then use them only once?

Reply:
Agreed, done